### ISSUE TITLE
[TypeError: 'type' object is not subscriptable]

### ISSUE THREAD

**Issue by weirdvic** [aliparlakci/bulk-downloader-for-reddit#299]
- [X] I am reporting a bug.

- [X] I am running the latest version of BDfR
- [ ] I have read the [Opening an issue](README.md#configuration)
  
## Description
After successful installation and running `python3 -m bdfr download` bdfr fails with an error.

## Command
```
python3 -m bdfr download
```

## Environment (please complete the following information):
 - OS: [Alpine Linux 3.12]
 - Python version: [3.8.8]

## Logs
```
$ python3 -m bdfr download
Traceback (most recent call last):
  File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/croesus/.local/lib/python3.8/site-packages/bdfr/__main__.py", line 8, in <module>
    from bdfr.archiver import Archiver
  File "/home/croesus/.local/lib/python3.8/site-packages/bdfr/archiver.py", line 15, in <module>
    from bdfr.archive_entry.submission_archive_entry import SubmissionArchiveEntry
  File "/home/croesus/.local/lib/python3.8/site-packages/bdfr/archive_entry/submission_archive_entry.py", line 13, in <module>
    class SubmissionArchiveEntry(BaseArchiveEntry):
  File "/home/croesus/.local/lib/python3.8/site-packages/bdfr/archive_entry/submission_archive_entry.py", line 40, in SubmissionArchiveEntry
    def _get_comments(self) -> list[dict]:
TypeError: 'type' object is not subscriptable
```

**Comment by weirdvic** [comment]
Nevermind, missed the python 3.9 is required part

**Comment by aliparlakci** [comment]
Addressed in #286
